.eslintrc.js

```diff
@@ -19,7 +19,7 @@ module.exports = {
     'flowtype',
   ],
   rules: {
-    'array-bracket-spacing': 'off', // use babel plugin rule
+    'array-bracket-spacing': ['error', 'never'],
```

Comment:
The 'babel/array-bracket-spacing' rule is deprecated and suggest to use 'array-bracket-spacing'